### DISCUSSION TITLE
Dependency graph doesn't recognize namespaced packages in package.json

### DISCUSSION THREAD

**Discussion by eps1lon** [community/community#21684] [New to GitHub]
<p>While the dependency graph (and the new “used by” feature) is working wonderfully for standard npm packages i.e. linking to the repos etc. it doesn’t fully connect namespaced packages.</p>
<p>Example:</p>
<p>Having <code>babel-core</code> in my package.json will list <code>babel-core</code> and link to babel/babel in the dependencies. However, <code>@babel/core</code> will not link to babel/babel even though it does list the dependency. Observable in e.g. <a href="https://github.com/emotion-js/emotion/network/dependencies" rel="nofollow noopener">https://github.com/emotion-js/emotion/network/dependencies or</a><a href="https://github.com/mui-org/material-ui/network/dependencies" rel="nofollow noopener">https://github.com/mui-org/material-ui/network/dependencies</a></p>
<p>This is especially an issue because namespaced packages will not have any dependents listed and will be considered as used by zero other repositories.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/eps1lon">@eps1lon</a>,</p>
<p>Thanks for being here and thanks for the feedback! I’ve passed it along with the team behind packages. I’m curious tho when you look up namespaced packages on <a href="https://www.npmjs.com/" rel="nofollow noopener">https://www.npmjs.com/</a> does it show the dependents?</p>

**Comment by eps1lon** [comment]
<p>&gt;  I’m curious tho when you look up namespaced packages on <a href="https://www.npmjs.com/" rel="nofollow noopener">https://www.npmjs.com/</a> does it show the dependents?</p>
<p><a class="mention" href="/andreagriffiths11">@andreagriffiths11</a>  Good question. For our particular case there are some listed (<a href="https://www.npmjs.com/package/@material-ui/core" rel="nofollow noopener">https://www.npmjs.com/package/@material-ui/core</a>) though we’re mostly used at an app level which aren’t published to npm. So I would assume npmjs is properly creating dependency graphs for namespaced packages</p>

**Comment by AndreaGriffiths11** [comment]
<p><a class="mention" href="/eps1lon">@eps1lon</a>, interesting! Thanks for the follow-up. I’ll be sure to pass the link along as well. </p>

**Comment by TremayneChrist** [comment]
<p>Hey <a class="mention" href="/andreagriffiths11">@andreagriffiths11</a> is there any update on this?</p>
<p>Many npm packages are now scoped/namespaced, which means more and more dependency links are breaking.</p>

**Comment by eps1lon** [comment, marked as answer]
<p>This seems to work now: </p>
<ul>
<li>
<a href="https://github.com/babel/babel/network/dependents" rel="nofollow noopener">https://github.com/babel/babel/network/dependents</a> </li>
</ul>
<p>- <a href="https://github.com/mui-org/material-ui/network/dependents" rel="nofollow noopener">https://github.com/mui-org/material-ui/network/dependents</a></p>
